src/materials/ConcreteThermalMoisture.C

```diff
@@ -377,19 +388,17 @@ PorousMediaBase::computeProperties()
         {
           case 0: // siliceous aggreagte
             if (T < 20.0)
-              _thermal_conductivity[qp] = _input_thermal_conductivity_of_concrete;
+              _thermal_conductivity[qp] = 1.4875;
             else if (T >= 20.0 && T < 800.0)
               _thermal_conductivity[qp] = -0.000625 * T + 1.5;
             else if (T >= 800.0)
               _thermal_conductivity[qp] = 1.0;
             break;
 
           case 1: // carbonate aggregate
-            if (T < 20.0)
-              _thermal_conductivity[qp] = _input_thermal_conductivity_of_concrete;
-            else if (T >= 20.0 && T < 293.0)
+            if (T < 293.0)
```

Comment:
This sounds suspiciously like 20 C in Kelvin.  Can you please verify that T in the formula is supposed to be in Celsius rather than Kelvin?